### PR TITLE
Netop.VisionTeacher version 1.7.6.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98692)

### CODE REVIEW

File: manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
```
[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.installer.yaml:4]: Please update file paths along with this change.

File: manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
```
Comment (stephengillie) [inline manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Netop.VisionTeacher
```

Comment (stephengillie) [reply manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml:4]: Please update file paths along with this change.

File: manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
```
Comment (stephengillie) [inline manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.yaml:4]:
```suggestion
PackageIdentifier: Netop.VisionTeacher
```

Comment (stephengillie) [reply manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.yaml:4]: Please update file paths to match this change.

File: manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
+PackageVersion: 1.7.6.0
+PackageLocale: en-US
+Publisher: Netop Solutions AS
```
Comment (stephengillie) [inline manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml:7]:
```suggestion
Publisher: Impero Solutions Ltd
```

Comment (stephengillie) [reply manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml:7]: impero bought Netop in January 2021. Should this be updated? 
https://www.imperosoftware.com/press-release/impero-software-acquires-netop-to-expand-its-global-education-and-corporate-product-offerings/

File: manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Neptop.VisionTeacher
+PackageVersion: 1.7.6.0
+PackageLocale: en-US
+Publisher: Netop Solutions AS
+PackageName: Vision Teacher for Chromebooks Machine-Wide Installer
```
Comment (stephengillie) [inline manifests/n/Neptop/VisionTeacher/1.7.6.0/Neptop.VisionTeacher.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.imperosoftware.com/
PackageName: Vision Teacher for Chromebooks Machine-Wide Installer
PackageUrl: https://landing.imperosoftware.com/chrome/win
```

**Comment by stephengillie** [comment]
Please update the file paths to match the changed ProductIdentifier. It still has the extra P. This is why validation is failing:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\n\Neptop\VisionTeacher\1.7.6.0', expected 'manifests\n\Netop\VisionTeacher\1.7.6.0'

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**